scss/_patterns_strip.scss

```diff
@@ -108,15 +108,6 @@ $color-suru-slant-left-fallback: rgba(205, 205, 205, 0.14) !default;
       linear-gradient(to bottom left, $color-suru-slant-right-fallback 0%, $color-suru-slant-right-fallback 49.8%, transparent 50%, transparent 100%),
       linear-gradient(to top right, #fff 0%, #fff 49%, transparent 50%, transparent 100%), linear-gradient(to top right, #fff 0%, #fff 100%),
       linear-gradient(111deg, $color-suru-start 10%, $color-suru-middle 37%, $color-suru-end 100%);
-
```

Comment:
@carkod Is this related to this PR at all? It seems it just moves the code around?

Reply:
This is the stylelint fix formatting...

Reply:
@carkod ok, thanks.

Could you remove it from this PR, it's not related and may later be confusing when looking why something was changed.
If you want feel free to push it separately in PR that fixes some of the warnings.